angular/core/components/email_settings_page/email_settings_page_controller.coffee

```diff
@@ -1,11 +1,7 @@
-angular.module('loomioApp').controller 'EmailSettingsPageController', ($rootScope, $translate, Records, AbilityService, FormService, Session, $location, ModalService, ChangeVolumeForm) ->
-  $rootScope.$broadcast('currentComponent', {page: 'emailSettingsPage'})
+angular.module('loomioApp').controller 'EmailSettingsPageController', (Records, AbilityService, FormService, Session, $location, ModalService, ChangeVolumeForm, $rootScope) ->
+  $rootScope.$broadcast('currentComponent', { page: 'emailSettingsPage'})
 
-  @init = =>
```

Comment:
I'm really nervous about this change; I'm pretty sure this is necessary. Did you test visiting with an unsubscribe token, as a logged out user, etc?